lib/SIL/SILVTable.cpp

```diff
@@ -33,27 +33,31 @@ SILVTable *SILVTable::create(SILModule &M, ClassDecl *Class,
   SILVTable *vt = ::new (buf) SILVTable(Class, Entries);
   M.vtables.push_back(vt);
   M.VTableMap[Class] = vt;
+  // Update the Module's cache with new vtable + vtable entries:
+  for (auto &entry : Entries) {
+    M.VTableEntryCache.insert(
+        std::make_pair(std::make_pair(vt, entry.first), entry.second));
```

Comment:
I think you can replace this with:

M.VTableEntryCache.insert({{vt, entry.first}, entry.second});

But my memory might be wrong.